spark/src/test/scala/org/apache/spark/sql/benchmark/CometReadBenchmark.scala

```diff
@@ -63,6 +65,7 @@ object CometReadBenchmark extends CometBenchmarkBase {
         sqlBenchmark.addCase("SQL Parquet - Comet") { _ =>
           withSQLConf(
             CometConf.COMET_ENABLED.key -> "true",
+            CometConf.COMET_EXEC_ENABLED.key -> "true",
```

Comment:
Why is this needed?

Reply:
> Why is this needed?

After #1483, we probably don't need this, I'll try to remove it later.

Reply:
CometBenchmarkBase sets COMET_EXEC_ENABLED to false by default, while native scan requires COMET_EXEC_ENABLED=true

https://github.com/apache/datafusion-comet/blob/main/spark/src/test/scala/org/apache/spark/sql/benchmark/CometBenchmarkBase.scala#L53-L54

https://github.com/apache/datafusion-comet/blob/52241085181449e4bdb3345b035bf28fef5b6c34/spark/src/main/scala/org/apache/comet/CometSparkSessionExtensions.scala#L213

Reply:
Line 68 is indeed redundant configuration, I will remove it.